tests/pytests/unit/proxy/test_napalm.py

```diff
@@ -0,0 +1,136 @@
+"""
+    :codeauthor: :email:`Anthony Shaw <anthonyshaw@apache.org>`
+"""
+
+
+import pytest
+
+import salt.proxy.napalm as napalm_proxy
+import tests.support.napalm as napalm_test_support
+from tests.support.mock import MagicMock, call, patch
+
+
+@pytest.fixture
+def test_opts():
```

Comment:
Perhaps a better name for the fixture, one that doesn't include the `test_` prefix.

Just so we don't stop to check what test is this 😄